app/views/users/show.html.erb

```diff
@@ -1 +1,15 @@
-<h1>Show specific user</h1>
\ No newline at end of file
+<section class="align">
+	<%= render 'shared/user', user: @user %>
+	<div class="align">
+		<div class="border">
+			<h2>Bio</h2>
+			<p class="bio"><%= @user.bio %></p>
+		</div>
+		<% @user.most_recent_posts.each do |post| %>
+		 <%= render 'shared/post', post:post %>
+		<%end %>
+	</div>
+	<div class="center">
```

Comment:
- [optional] Great job.Though everything works well always follow the wireframe design as much as possible. you can centralize the buttons, like the `see all post` and `paganization` so they follow the wireframe. nice work.